### ISSUE TITLE
Use translator for all strings

### ISSUE THREAD

**Issue by haxxmaxx** [qlik-oss/sn-table#289]
When we have the strings merged, we need to make sure that we use the translator for all places. Right now they are missing in the pagination and for screen readers mainly. By searching for all the values (e.g. `value deselected`) we can find the ones that are not using translator
